src/data/completedApplication.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Contains the data for an example completed loan application.  This data is valid from an API standpoint
+ * and is used by the devtools for auto-completing the application during development.
+ */
+
+export default {
+  selected: {
+    savings: 'UNSELECTED',
+    spending: 'SELECTED',
+    credit: 'UNSELECTED'
+  },
+  router: {
+    location: {
+      pathname: '/apply',
+      search: '',
+      hash: '',
+      key: '3pixpz'
+    }
+  },
+  form: {
```

Comment:
I'd rather not directly mess with redux-form state like this. Consider using the actions built into redux form instead